### PR TITLE
Ada1 50

### PR DESCRIPTION
**Pull request by GabrielParle**
bug fixed

### CODE REVIEW

File: AgentDiscoveries-Frontend/app/src/components/admin/location-form.jsx [hunk 1]
```diff
@@ -96,6 +121,19 @@ export default class LocationForm extends React.Component {
         );
     }
 
+    // renderifTrue(){
```
Comment (Astra2707) [inline AgentDiscoveries-Frontend/app/src/components/admin/location-form.jsx:124]: Maybe remove comment?

**Review by Astra2707 — APPROVED** [review]

File: AgentDiscoveries-Frontend/app/src/components/admin/location-form.jsx [hunk 2]
```diff
@@ -24,27 +25,37 @@ export default class LocationForm extends React.Component {
...
             this.loadLocation(this.props.id);
         }
     }
 
+    componentDidMount() {
+        this.loadRegions();
+    
+    }
+    
+
```
Comment (oscarmccormack) [inline AgentDiscoveries-Frontend/app/src/components/admin/location-form.jsx:41]: blank lines

**Review by oscarmccormack — COMMENTED** [review]
Looks good to me delete the comment that josh commented about

**Comment by GabrielParle** [comment]
pushed changes removed blank space and comments

**Pull request merged**